CONTRIBUTING.md

```diff
@@ -1,65 +1,69 @@
-
-5. Determine if any critical NCL computational functions are missing. If so, please submit a new issue (or comment on an existing issue) on this repository's GitHub page containing the script name and function name, select a new NCL script, and start this process over.
-
-6. Determine if any data files needed by your script are available in the [geocat-datafiles](https://github.com/NCAR/GeoCAT-datafiles) repository:
-
-   ```python
-   >>> # Determine if data files of interest are available:
-   >>> import geocat.datafiles
-   >>> print(geocat.datafiles.POOCH.registry_files)
-   ```
-
-   If not, please submit a new issue (or comment on an existing issue) on this repository's GitHub page containing the name of the missing datafile(s).
-
-7. Create a new Python script using the original NCL script's basename, but prepended with "NCL_". For example:
-    `new_script_1.ncl` becomes `NCL_new_script_1.py`
-
-    * Note that `template_script.py` at the root of this repository is a great starting point for a new script, so you may want to consider copying that to `NCL_new_script_1.py` instead of starting from scratch.
-
-
-8. The general objective of this project is to identify any NCL plotting functionality that is missing from the popular Matplotlib + Cartopy toolchain, so each contributed script should contain a best-effort attempt at reproducing an NCL graphic as closely as possible without using NCL or PyNGL. In the future, PyNGL examples may be included in this repository as well, but the current goal is to see what *can't* be done with Matplotlib and Cartopy.
-
-   That said, if you identify any NCL functionality in particular that does not seem to exist in Matplotlib/Cartopy, please create an issue on this repository's GitHub page with a link to the original NCL example page and list any NCL functions whose functionality is missing from Python.
-
-9. When you have completed an example script, please try building the sphinx documentation from this repository using "make html" from the repository's root directory. Then open the HTML file located at `_build/html/gallery/index.html` and try to find your new example, ensuring that any output (text or graphical) matches what you expected.
+Please first refer to [GeoCAT Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) for overall 
+contribution guidelines (such as detailed description of GeoCAT structure, forking, repository cloning, 
+branching, etc.). Once you determine that a function should be contributed under this repo, please refer to the 
+following contribution guidelines:
+
+
+# Adding new plotting scripts to the Geocat-examples repo
+
+1. Please check the followings to ensure that the example you are about to work on has not been ported yet:
+    
+    - [GeoCAT-Examples Gallery](https://geocat-examples.readthedocs.io/en/latest/) or 
+    the `Plots` directory of this repo,
```

Comment:
Will computational examples go in the Plots directory?

Reply:
No. To address, added "GeoCAT-comp" statement in place. FYI: Such a folder with extended examples will be soon created in response to a corresponding issue.